book/10-git-internals/sections/objects.asc

```diff
@@ -328,7 +328,7 @@ $ find .git/objects -type f
 
 If you follow all the internal pointers, you get an object graph something like this:
 
-.All the objects in your Git directory.
+.All the reachable objects in your Git directory.
 image::images/data-model-3.png[All the objects in your Git directory.]
```

Comment:
It's possible, but not necessary. There are some projects on GitHub that require every commit to be pristine, but this is not one of them. :)